Userland/Services/SystemServer/main.cpp

```diff
@@ -32,6 +32,24 @@
 String g_system_mode = "graphical";
 NonnullRefPtrVector<Service> g_services;
 
+static AK::String determine_single_app_mode()
```

Comment:
duplicate?

Reply:
While it is, I'd rather not touch Lib* for a function that'll be used only 2 times in the entire codebase.